packages/bpk-component-barchart/src/utils.js

```diff
@@ -16,24 +16,26 @@
  * limitations under the License.
  */
 
-const center = scale => {
+/* @flow strict */
+
+const center = (scale: Function) => {
   let offset = scale.bandwidth() / 2;
   if (scale.round()) {
     offset = Math.round(offset);
   }
-  return d => scale(d) + offset;
+  return (d: number) => scale(d) + offset;
 };
 
-const identity = x => x;
+const identity = (x: any) => x;
```

Comment:
We typically try to avoid `any` unless absolutely necessary, is there something more specific we can use here?